gometa.go

```diff
@@ -29,6 +31,10 @@ func gometa(w http.ResponseWriter, r record, host, path string) error {
 	if path == "/" {
 		path = ""
 	}
+	bl := "/internal"
```

Comment:
This is potentially me missing a really obvious thing, but I don't get what `bl` stands for :sweat_smile: depending on how obvious/non obvious it is we might want to change the naming ?

Reply:
blacklist.